routers/repo/issue.go

```diff
@@ -48,7 +48,9 @@ func MustEnableIssues(ctx *middleware.Context) {
 }
 
 func MustEnablePulls(ctx *middleware.Context) {
```

Comment:
@Unknwon You take clean code very seriously. :)
Fixed in a second commit. Thanks for the feedback.